tests/shared/test-utils/wp-element-mock.js

```diff
@@ -0,0 +1,112 @@
+/**
```

Comment:
This goes in a new `shared` folder, as it will also be needed for the integration tests, that I'm working on elsewhere.